spec/donation_system/stripe_wrapper/input_data_validator_spec.rb

```diff
@@ -1,32 +1,26 @@
 # frozen_string_literal: true
 
+require 'donation_system/data_structs_for_tests'
 require 'donation_system/stripe_wrapper/input_data_validator'
 require 'spec_helper'
 
 module DonationSystem
   module StripeWrapper
     RSpec.describe InputDataValidator do
-      RawInputData = Struct.new(:amount, :currency, :token, :email, :name)
-
-      let(:data) do
-        RawInputData.new(
-          '10.50', 'usd', 'stripe_token', 'user@example.com', 'Name'
-        )
-      end
-      let(:result) { validate(data) }
-      let(:fields) { result.item }
-
       describe 'Stripe required fields' do
+        let(:result) { validate(VALID_REQUEST_DATA) }
+        let(:fields) { result.item }
+
         it 'has required field amount' do
-          expect(fields[:amount]).to eq(1050)
+          expect(fields[:amount]).to eq(1234)
         end
 
         it 'has required field currency' do
-          expect(fields[:currency]).to eq('usd')
+          expect(fields[:currency]).to eq('gbp')
         end
 
         it 'has required field Stripe token' do
-          expect(fields[:source]).to eq('stripe_token')
+          expect(fields[:source]).to eq('tok_visa')
```

Comment:
rename to `token_visa`?

Reply:
![test](https://user-images.githubusercontent.com/2157089/32961522-c5e752a8-cbc0-11e7-9f64-18f63c5fe6e0.png)